CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 3.17.0
+
+- Change the Docxtemplater constructor to accept zip and modules as part of arguments. It is now possible to call `new Docxtemplater(zip)` instead of `new Docxtemplater().loadZip(zip)`
```

Comment:
Please replace : 

> - Change the Docxtemplater constructor to accept zip and modules as part of arguments. It is now possible to call `new Docxtemplater(zip)` instead of `new Docxtemplater().loadZip(zip)`
> - This is a backward compatible change

to : 

> - Add constructor method that accepts zip and optionally modules. This constructor will be, in docxtemplater v4, the official constructor, and the methods : `loadZip`, `attachModule` and `compile` will no more be available.
> - This change is backward compatible, meaning that you can continue to use the constructor with no arguments at the time being.